src/screens/FeedScreen/RenderList.js

```diff
@@ -14,16 +14,17 @@ import {
   SOURCE_FEED_TAB
 } from '../../utils/constants';
 import {getCommentLength} from '../../utils/getstream';
-import {normalizeFontSizeByWidth} from '../../utils/fonts';
+import {normalize, normalizeFontSizeByWidth} from '../../utils/fonts';
 import {COLORS} from '../../utils/theme';
 import dimen from '../../utils/dimen';
-
 import ShareUtils from '../../utils/share';
 import Content from './Content';
 import ContentLink from './ContentLink';
 import Header from './Header';
 import useFeed from './hooks/useFeed';
 import usePostHook from '../../hooks/core/post/usePostHook';
+import WriteComment from '../../components/Comments/WriteComment';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import statement for `normalize` is added but it's not used anywhere in the provided code. If it's not used in other parts of the file, consider removing this import to keep the code clean and maintainable.
```diff
- import {normalize, normalizeFontSizeByWidth} from '../../utils/fonts';
+ import {normalizeFontSizeByWidth} from '../../utils/fonts';
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->